src/mds/MDCache.cc

```diff
@@ -7437,6 +7437,7 @@ void MDCache::check_memory_usage()
 	   << ", " << Capability::count() << " caps, " << caps_per_inode << " caps per inode"
 	   << dendl;
 
+  mds->update_mlogger();
```

Comment:
i think it would be more proper here, since another two member of mds->mlogger (rss and heap) are updated right here.